lms/djangoapps/courseware/tests/test_video_mongo.py

```diff
@@ -1252,7 +1253,7 @@ def test_editor_saved_with_yt_val_profile(self, default_store):
 
 
 @ddt.ddt
-class TestVideoDescriptorStudentViewJson(TestCase):
+class TestVideoDescriptorStudentViewJson(CacheIsolationTestCase):
```

Comment:
**Note to reviewers:** The need for this change confirms that the video_module's caching wasn't actually happening before.  Hopefully, this will result in better performance for our [Courseware page](https://rpm.newrelic.com/accounts/88178/applications/3343327/transactions#id=5b225765625472616e73616374696f6e2f46756e6374696f6e2f636f75727365776172652e76696577732e696e6465783a436f7572736577617265496e6465782e676574222c22225d) and [edxval queries](https://rpm.newrelic.com/accounts/88178/applications/3343327/datastores#/overview/MySQL/drilldown?metric=Datastore%252Fstatement%252FMySQL%252Fedxval_profile%252Fselect&value=total_call_time_per_minute).